tests/python/test_tracepoint.py

```diff
@@ -64,5 +64,44 @@ def test_tracepoint_data_loc(self):
         self.assertTrue("/bin/ls" in [v.filename.decode()
                                       for v in b["execs"].values()])
 
+@unittest.skipUnless(kernel_version_ge(4,7), "requires kernel >= 4.7")
+class TestTracepointDataLoc(unittest.TestCase):
+    def test_tracepoint_data_loc(self):
```

Comment:
class name TestTracepointDataLoc has been used before and you need to choose a different name.